src/Microsoft.Health.Dicom.Core/Exceptions/WorkitemIsInFinalStateException.cs

```diff
@@ -10,8 +10,11 @@ namespace Microsoft.Health.Dicom.Core.Exceptions;
 /// </summary>
 public class WorkitemIsInFinalStateException : DicomServerException
 {
-    public WorkitemIsInFinalStateException(string workitemUid, string procedureStepState)
-        : base(string.Format(DicomCoreResource.WorkitemIsInFinalState, workitemUid, procedureStepState))
+    public WorkitemIsInFinalStateException(string workitemUid, string procedureStepState, ushort errorOrWarningCode)
+        : base(string.Format(DicomCoreResource.WorkitemIsInFinalState, workitemUid, procedureStepState, errorOrWarningCode))
     {
+        ErrorOrWarningCode = errorOrWarningCode;
     }
+
+    public ushort ErrorOrWarningCode { get; }
```

Comment:
Should we summarize/comment here on what value is considered an error vs. warning?
As an alternative, we can mark this as a boolean and call it `IsWarning`. If this is set to false, this exception will be considered as an error.

Reply:
This type is totally unused for this PR. I'll remove it.